lang/mcache/utils.go

```diff
@@ -14,6 +14,12 @@
 
 package mcache
 
+import "math/bits"
+
+func bsr(x int) int {
+	return bits.Len64(uint64(x)) - 1
```

Comment:
Maybe we could use `bits.Len(uint(x))`  instead of `bits.Len64(uint64(x))`? so we can use this function for both 32-bit and 64-bit platforms.

Reply:
solved